examples/chalice/blueprints/authed.py

```
@@ -3,7 +3,11 @@
 
 from chalice import Blueprint
 
-from ...config import TEST_DEFAULT_PLATFORM_ID, TEST_DEFAULT_USER_ID
+from ...config import (
+    TEST_DEFAULT_API_KEY_ID,
+    TEST_DEFAULT_PLATFORM_ID,
+    TEST_DEFAULT_USER_ID,
+)
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)